services/QuillLMS/app/serializers/lesson_planner/unit_serializer.rb

```diff
@@ -73,7 +73,7 @@ class LessonPlanner::UnitSerializer < ActiveModel::Serializer
   end
 
   private def dueDates
-    object.reload.unit_activities.uniq(&:activity).each_with_object({}) do |unit_activity, acc|
```

Comment:
Was the reload here messing up the touches?

Reply:
This `reload` was causing an ordering problem. This reload causes this (it was just surfaced by adding the touches):
- Old object used by: "methods called before `dueDates` is called"
- New object used by: "`dueDates` and any methods called after it OR all methods if you call `as_json` a second time"

As an aside, I couldn't find this serializer used anywhere, but there's the gem has some implicit usage by path/file/model name, so I'm not 100% sure, so I haven't removed it yet.